packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -87,6 +93,27 @@ const HeaderTools = ({
                 document.querySelector('html').classList.toggle('pf-v5-theme-dark')} />
             </ToolbarItem>
           )}
+          {hasRTLSwitcher && (
+            <ToolbarItem>
+              <ToggleGroup aria-label="Example direction toggle">
+                <ToggleGroupItem
+                  icon={isRtl ? <AlignRightIcon /> : <AlignLeftIcon />}
+                  aria-label="Toggle direction"
+                  buttonId="ws-rtl-toggle"
+                  isSelected={false}
+                  onChange={() => setIsRtl(isRtl => {
+                    const examples = document.querySelectorAll('.ws-example');
+                    examples.forEach(ex => ex.classList.toggle('pf-v5-m-dir-rtl'));
```

Comment:
also not a big deal but when these vars are avialble in react tokens, you can use the token instead so you don't have to hardcode it (and update the `v5` for v6)

Reply:
Great callout! ➕➕